flytekit/clients/friendly.py

```diff
@@ -1046,3 +1048,33 @@ def get_data(self, flyte_uri: str) -> _data_proxy_pb2.GetDataResponse:
 
         resp = self._dataproxy_stub.GetData(req, metadata=self._metadata)
         return resp
+
+    def get_download_deck_signed_url(
```

Comment:
Do you think we need to change the function name to `get_download_signed_url` and add `ArtifactType` to the input args?
cc @wild-endeavor @eapolinario

Reply:
yeah. What if we set the default `artifact_type` to `ARTIFACT_TYPE_DECK`?

Reply:
Yes I can do that